terraform/e2e/gclb.tf

```diff
@@ -0,0 +1,219 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_compute_global_address" "default" {
```

Comment:
Why not use this module? https://registry.terraform.io/modules/GoogleCloudPlatform/lb-http/google/latest/submodules/serverless_negs

Reply:
I think the question should be "why use". What benefit does the module bring? Complicated modules usually require learnings to use and will stop working as soon as we need to turn a knob it doesn't support (e.g. as an input).

Reply:
Ok, not a blocker. One benefit I can think of is it hides the technical details, only need to care about input and output.